config/locales/en.yml

```diff
@@ -6296,6 +6296,8 @@ en:
   view_more_observations_of_x_project: "View more observations from %{project} on <nobr>%{site_name} &raquo;</nobr>"
   view_more_observations_of_x_taxon: "View more observations of %{taxon} on <nobr>%{site_name} &raquo;</nobr>"
   view_more_unreviewed: "View more unreviewed"
+  # Link to view identifications affected by a taxon change
+  view_my_affected_identifications: View my affected identifications
```

Comment:
Argh, sorry I didn't notice this before: we generally avoid using the first person in UI text like this, so please change this to "View your affected IDs"